### PR TITLE
📝 (post) Add Web Apps on Google Play Post

### PR DESCRIPTION
**Pull request by fuentespatrick**
**Please describe what your Pull Request does**
Add the blog post for web apps on Google Play

### CODE REVIEW

File: pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+![A laptop showing a Google Play Billing subscription checkout flow on Clipchamp](/images/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play/play_billing_subscription.png)
+
+## More amazing apps coming to Chrome OS
+
+We’ve already seen [great responses to PWAs on Chromebooks](https://blog.google/products/chromebooks/whats-new-chrome-os/dec2020/) — users enjoying the advanced graphics on [Adobe Spark](https://spark.adobe.com/) and [Corel’s Gravit Designer](http://web.dev/gravit-designer), sharper video editing with [Clipchamp](http://web.dev/clipchamp), and more powerful viewing experiences on [YouTube TV](https://tv.youtube.com/). We’re excited to share that all of these experiences will arrive in the Play Store as soon as the Digital Goods API arrives in Chrome OS 89.
+
+It’s incredible to see the journey of developers as they build their web apps for larger screens. We can’t wait to showcase amazing web apps like these on Google Play — and we hope to see yours there soon!
+
+[^1]: The NPD Group, Inc., U.S. Retail Tracking Service, Notebook Computers, based on unit sales, Jan.–Aug. 2020.
```
Comment (joycetoh8) [inline pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md:48]: The footnotes are totally fine at the end of the page. But I think Sam previously recommended putting them right after the reference (in this case after line 22 and before the next lines) so it's easier to find for anyone looking at the raw file in the future. This doesn't affect how it'll actually get displayed on chromeos.dev, it'll still appear at the end. But just an fyi

File: pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Search continues to be a quick and easy way to discover new web apps, but many people are also turning to app stores to find the best software for their device in one place. That’s why we gave developers the ability to list their PWA in Google Play on Chrome OS using [Trusted Web Activity](https://developers.google.com/web/android/trusted-web-activity). Now, users can discover web apps by browsing through collections and curated recommendations in the Play Store. And once they install it, they’ll be able to enjoy the full-screen app experience they love, powered by Chrome.
+
+Brands like [Kapwing](https://www.kapwing.com/), an online image, video, and GIF editing platform, have already seen impressive results after launching their PWA on the Play Store:
+
+![Quote from Julia Enthoven, CEO of Kapwing: "In the first five weeks of launching our PWA-ified website, the number of people creating videos through the PWA grew 36%. This growth in usage outpaced overall growth on the website, indicating higher retention among creators who installed the PWA."](/images/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play/kapwing_quote.png)
```
Comment (joycetoh8) [inline pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md:32]: If you want to match the quote styling that's already on chromeos.dev, you can replace this image with
```suggestion
> In the first five weeks of launching our PWA-ified website, the number of people creating videos through the PWA grew 36%. This growth in usage outpaced overall growth on the website, indicating higher retention among creators who installed the PWA.

Julia Enthoven, CEO of Kapwing {.cite}

```

**Review by joycetoh8 — APPROVED** [review]

File: pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md (hunk 2 above)
Comment (joycetoh8) [reply pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md:32]: For a reference of what that'll look like: https://chromeos.dev/en/stories/concepts#results

File: pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md (hunk 1 above)
Comment (fuentespatrick) [reply pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md]: Thanks, I just pushed a change addressing this feedback

File: pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md (hunk 2 above)
Comment (fuentespatrick) [reply pages/en/posts/easy-to-build-monetize-and-discover-list-your-web-app-on-google-play.md]: Thanks, I just pushed a change addressing this feedback

**Review by joycetoh8 — APPROVED** [review]
Looks great, thanks!!

**Pull request merged**